source/_includes/common-tasks/data_disk.md

```diff
@@ -64,3 +64,42 @@ The system will prepare the data disk and immediately reboot. The reboot will ta
 Using an USB attached SSD can draw quite some power. For instance on Raspberry Pi 3 the official Raspberry Pi power supply (PSU) only provides 2.5A which can be too tight. Use a more powerful power supply if you experience issues. Alternatively use a powered USB hub. Connect the Hub to one of the USB slots of your Raspberry Pi, and connect the SSD to the Hub. In this setup the power supply of the Hub will power the attached device(s).
 
 </div>
+
+### Migrating an external data disk to another system
+
+This section shows how to move an external data disk from one system to another.
+This can be an option if the following elements apply to your use case:
+
+- You already have a functioning Home Assistant instance (system 1) that is using an external data disk.
+- You have another, new, functioning Home Assistant instance (system 2).
+- You want now want to use the data disk of system 1 on system 2 instead.
+
+The aim is to migrate the data from system 1 to system 2. One way to do this is by [restoring a backup](/common-tasks/os/#restoring-a-backup). The other way is to move the data disk. This can be an interesting option if you have a large amount of data on your external disk or if your external disk has more storage capacity than your new system.
+
+#### Prerequisites
+
+- Home Assistant Operating System 13.0 or later
```

Comment:
13 does not yet exist?
https://github.com/home-assistant/operating-system/releases

Generally we do not note version numbers, as that is done here, should it mention core 2024.5.0+?